apps/services/auth/admin-api/src/app/v2/secrets/client-secrets.service.ts

```diff
@@ -99,22 +101,40 @@ export class ClientSecretsService {
     return Boolean(client)
   }
 
-  formatSecret(secret: ClientSecret): ClientSecretDto {
+  private formatSecret(secret: ClientSecret): ClientSecretDto {
     if (environment.clientSecretEncryptionKey === undefined) {
```

Comment:
We should be able to remove this check as the constructor will throw error initializing the class instance if the environment variable is not set.

Reply:
Missed that, removed.